src/redwood.ts

```diff
@@ -0,0 +1,914 @@
+    },
+    {
+      name: "info",
+      icon,
+      description: "Print your system environment information",
+    },
+    {
+      name: "upgrade",
+      icon,
+      description: "Upgrade all @redwoodjs packages via an interactive CLI",
+      options: [
+        {
+          name: ["--dry-run", "-d"],
+          description: "Check for outdated packages without upgrading",
+        },
+        {
+          name: ["--tag", "-t"],
+          description:
+            'Choices are "canary", "rc", or a specific version (e.g. "0.19.3"). WARNING: Unstable releases in the case of "canary" and "rc", which will force upgrade packages to the most recent release of the specified tag',
+        },
+        {
+          name: "--pr",
+          description: "Installs packages for the given PR",
+        },
+      ],
+    },
+    {
+      name: "serve",
+      icon,
+      description: "Runs a server that serves both the api and the web sides",
+      args: {
+        name: "side",
+        description: "Which side(s) to check (Default to both)",
+        isOptional: true,
+        suggestions: [
+          {
+            name: "api",
+            description: "Check the api server",
+          },
+          {
+            name: "web",
+            description: "Check the web server",
+          },
+        ],
+      },
+      options: [
+        {
+          name: "--port",
+          description: "What port should the server run on [default: 8911]",
+        },
+        {
+          name: "--socket",
+          description:
+            "The socket the server should run. This takes precedence over port",
+        },
+        {
+          name: "--apiRootPath",
+          description: "The root path where your api functions are served",
+        },
+        {
+          name: "--apiHost",
+          description:
+            "Forwards requests from the apiProxyPath (defined in redwood.toml) to the specified host",
+        },
+      ],
+    },
+    {
+      name: ["type-check", "tsc", "tc"],
+      icon,
+      description:
+        "Runs a TypeScript compiler check on both the api and the web sides",
+      args: {
+        name: "side",
+        description: "Which side(s) to check (Default to both)",
+        isOptional: true,
+        suggestions: [
+          {
+            name: "api",
+            description: "Check the api server",
+          },
+          {
+            name: "web",
+            description: "Check the web server",
+          },
+        ],
+      },
+    },
+    {
+      name: "test",
+      icon,
+      description: "Run Jest tests for api and web",
+      args: {
+        name: "sides or filter",
+        description:
+          "Which side(s) to test, and/or a regular expression to match against your test files to filter by",
+        isOptional: true,
+        suggestions: [
+          {
+            name: "api",
+          },
+          {
+            name: "web",
+          },
+        ],
```

Comment:
In those cases where each suggestion only have a name, we can simply use an array of string:
```suggestion
        suggestions: [
           "api",
          "web",
        ],
```